packages/logger/src/helpers/helper-simple-deep-clone.ts

```
@@ -0,0 +1,51 @@
+ */
+export function simpleDeepClone<T extends string | number | any[] | object>(
+  input: T
+): T {
+  // if not array or object or is null return self
+  if (typeof input !== 'object' || input === null) {
+    return input;
+  }
+  let output: any;
+  let i: number | string;
+  // handle case: array
+  if (input instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)